scripts/metricsgen/metricsgen.go

```diff
@@ -280,7 +296,11 @@ func extractLabels(bl *ast.BasicLit) string {
 	if bl != nil {
 		t := reflect.StructTag(strings.Trim(bl.Value, "`"))
 		if v := t.Get(labelsTag); v != "" {
-			return v
+			var res string
+			for _, s := range strings.Split(v, ",") {
+				res += strconv.Quote(strings.TrimSpace(s)) + ","
+			}
+			return res
```

Comment:
Optional suggestion:
```suggestion
			var res []string
			for _, s := range strings.Split(v, ",") {
				res = append(res, strconv.Quote(strings.TrimSpace(s)))
			}
			return strings.Join(res, ",")
```